src/main/docs/guide/ioc/introspection/atIntrospected.adoc

```diff
@@ -11,3 +11,7 @@ snippet::io.micronaut.docs.ioc.beans.IntrospectionSpec[tags="usage", indent=0]
 <3> A api:core.beans.BeanProperty[] can be retrieved from the introspection
 <4> Use the `set` method to set the property value
 <5> Use the `get` method to retrieve the property value
+
+It's possible to apply the introspection to all the classes in one package. Simply create `package-info` file and annotate the package with ann:core.annotation.Introspected[].
+
+snippet::io.micronaut.docs.ioc.introspection.pck.foobar.package-info[tags="usage", indent=0]
```

Comment:
add a `WARNING:` here that the package should only contain classes that can be introspected. Also clarify here whether this behaviour is recursive for sub packages or a new package-info is required for each.